sympy/geometry/tests/test_polygon.py

```diff
@@ -228,6 +228,11 @@ def test_polygon():
     assert intersection(m[p1], m[p2], m[p3]) == [t1.centroid]
     assert t1.medial == Triangle(Point(2.5, 0), Point(0, 2.5), Point(2.5, 2.5))
 
+    # Nine-point circle
+    vo_middles = [Segment(v, t1.orthocenter).midpoint for v in t1.vertices]
```

Comment:
``vo_middles`` in this case is ``[Point2D(2, 0), <bound method Point2D.midpoint of Point2D(4, 0)>, Point2D(4, 3/2)]``, probably due to t1 being a right-angle triangle.